consensus/chronology/chronology_test.go

```diff
@@ -36,6 +38,8 @@ func TestChronology_NewChronologyNilRounderShouldFail(t *testing.T) {
 		genesisTime,
 		nil,
 		syncTimerMock,
+		alarm.NewAlarmScheduler(),
+		make(chan endProcess.ArgEndProcess),
```

Comment:
make this a buffered chan in order to finish the associated go routines as fast as it can. (otherwise the go routines will wait until the package finish all tests)

Reply:
Refactored.